commands/out.go

```diff
@@ -245,11 +256,47 @@ func createAuth(req resource.OutRequest) *authn.Basic {
 	}
 }
 
+func createTransport(req resource.OutRequest) (http.RoundTripper, error) {
```

Comment:
Maybe we can extract this into a method on `Source`? It looks to be duplicated at the moment with `types.go`.

Actually not sure why the code doesn't just use the `(Source).AuthOptions` method here too, but I haven't looked too closely. 🤔

Reply:
I updated the `out` command to use a method from `Source`.